source/upgrade/upgrade_notes.rst

```diff
@@ -20,18 +20,37 @@ Upgrade notes
 * ``wazo-dird`` phone plugins have been migrated to ``wazo-phoned``. If you used the phone routes
   from ``wazo-dird`` directly, you must use the new routes in ``wazo-phoned``.
 
-* The conference rooms created with the old orange web interface (using asterisk ``meetme`` module)
-  will not work anymore because they rely on Dahdi. If you were still using them, you must create
-  new conference rooms using the conferences API of the ``wazo-ui`` interface.
-
-* ``dahdi`` is not longer a dependency on Wazo. Upgraded engines will include all dependencies to
-  have a working dahdi installation. If you do not want dahdi installed on your system execute the
-  following steps::
-
-    rm -f /etc/wazo-confgend/conf.d/50_enable_chan_dahdi.yml
-    systemctl restart wazo-confgend
-    # THIS WILL RESTART ASTERISK
-    apt remove wazo-asterisk-extra-modules
+* The conference rooms created in Wazo 18.03 or before (using asterisk ``meetme`` module) will not
+  work anymore because they rely on DAHDI. If you were still using those conference rooms, you must
+  create new conference rooms using the conferences API or the ``wazo-ui`` interface.
+
+* ``DAHDI`` is not longer a mandatory dependency of Wazo: it will not be installed on new installs
+  anymore. Upgraded Wazo Platform will keep DAHDI installed if it was configured in
+  ``/etc/asterisk/dahdi_channels.conf``. Otherwise, DAHDI will be removed. To install or remove
+  DAHDI manually, see :ref:`chan_dahdi`. Other disabled Asterisk modules:
+
+  * ``app_jack``
+  * ``cdr_pgsql``
```

Comment:
I'm not sure if listing disabled modules is useful. chan_dahdi is the only one that was previously enabled that is now disabled and many more modules are disabled.

```
enabled_asterisk_modules:
  app_adsiprog.so: false
  app_alarmreceiver.so: false
  app_amd.so: false
  app_cdr.so: false
  app_dictate.so: false
  app_externalivr.so: false
  app_festival.so: false
  app_followme.so: false
  app_forkcdr.so: false
  app_ices.so: false
  app_jack.so: false
  app_minivm.so: false
  app_morsecode.so: false
  app_mp3.so: false
  app_nbscat.so: false
  app_speech_utils.so: false
  app_test.so: false
  cdr_adaptive_odbc.so: false
  cdr_csv.so: false
  cdr_custom.so: false
  cdr_manager.so: false
  cdr_odbc.so: false
  cdr_pgsql.so: false
  cdr_radius.so: false
  cdr_sqlite3_custom.so: false
  cdr_syslog.so: false
  cdr_tds.so: false
  cel_custom.so: false
  cel_pgsql.so: false
  cel_radius.so: false
  cel_sqlite3_custom.so: false
  cel_tds.so: false
  chan_alsa.so: false
  chan_console.so: false
  chan_dahdi.so: false
  chan_mgcp.so: false
  chan_motif.so: false
  chan_oss.so: false
  chan_phone.so: false
  chan_sip.so: false
  chan_skinny.so: false
  chan_unistim.so: false
  codec_dahdi.so: false
  func_cdr.so: false
  func_odbc.so: false
  pbx_ael.so: false
  pbx_loopback.so: false
  pbx_lua.so: false
  res_ari_mailboxes.so: false
  res_calendar.so: false
  res_calendar_caldav.so: false
  res_calendar_ews.so: false
  res_calendar_exchange.so: false
  res_calendar_icalendar.so: false
  res_clialiases.so: false
  res_config_ldap.so: false
  res_config_pgsql.so: false
  res_config_sqlite.so: false
  res_config_sqlite3.so: false
  res_mwi_external.so: false
  res_phoneprov.so: false
  res_pjsip_config_wizard.so: false
  res_snmp.so: false
  res_stasis_mailbox.so: false
  res_statsd.so: false
  res_stun_monitor.so: false
  res_xmpp.so: false
```

Reply:
Ah, thanks. I think some people enable non-Wazo modules (i.e. modules not enabled by default on Wazo Platform), so I'd like to give them the list of modules that may be affected by the upgrade. I guess this list comes from `xivo-confgen asterisk/modules.conf`? Can we add an upgrade note if non-Wazo modules can get disabled by the upgrade?